app/views/admin/shared/_nested_sidebar.html.erb

```diff
@@ -0,0 +1,36 @@
+<% menu_items.each do |group_name, group| %>
+  <li>
+    <% if group[:children].length == 1 %>
+      <a class="crayons-link crayons-link--block <%= "crayons-link--current" if deduced_controller(request) == group[:children][0][:controller] %>"
+         href="/admin/<%= group[:children][0][:controller] %>"
+         aria-page="<%= "page" if deduced_controller(request) == group[:children][0][:controller] %>"
+      >
+        <%= inline_svg_tag("#{group[:svg]}", aria: true, class: "dropdown-icon crayons-icon") %>
+        <%= display_name(group_name) %>
+      </a>
+    <% else %>
+      <button class="crayons-link crayons-link--block cursor-pointer <%= "crayons-link--current" if deduced_scope(request) == group_name.to_s %>"
+        data-toggle="collapse"
+        data-target="#<%= group_name %>"
+        aria-expanded="<%= (deduced_scope(request) == group_name.to_s).to_s %>"
+        aria-controls="<%= group_name %>"
+      >
+        <%= inline_svg_tag("#{group[:svg]}", aria: true, class: "dropdown-icon crayons-icon", title: "#{group_name}") %>
+        <%= display_name(group_name) %>
+      </button>
+      <ul id="<%= group_name %>"
+           class="<%= (deduced_scope(request) == group_name.to_s) ? "expand show" : "collapse hide" %>" aria-labelledby="<%= group_name %>">
+        <% group[:children].each do |item| %>
+          <li>
+            <a class="crayons-link crayons-link--block ml-7 <%= "fw-bold" if deduced_controller(request) == item[:controller].to_s %>"
+              href="/admin/<%= group_name %>/<%= item[:controller] %>"
+              aria-page="<%= "page" if deduced_controller(request) == item[:controller].to_s %>"
+            >
+              <%= item[:name].to_s.titleize %>
```

Comment:
I may be looking too hard for patterns here, but the single-item special case above uses the `display_name` helper method. Should we use that here, as well?

```suggestion
              <%= display_name(item[:name]) %>
```

Reply:
@jgaskins no it wouldn't and here's the reason:

`group_name` is the name of a group like `:content_manager` - it's the first outer key in the payload, hence we need to manipulate it using `display_name`  by replacing underscores with spaces and using titleize on it. 

whereas the `item[:name]` is the keys inside the payload for the groups navigation items. Since we do not always want to use the controller_names for these items, we specify names for the navigation items in the payload which already include the spaces. Hence we just make sure it's titleized. Hope this makes sense, feel free to ask any follow up questions